django/db/backends/mysql/operations.py

```diff
@@ -232,10 +232,17 @@ def get_db_converters(self, expression):
         elif internal_type == 'DateTimeField':
             if settings.USE_TZ:
                 converters.append(self.convert_datetimefield_value)
+        elif internal_type == 'CharField':
+            converters.append(self.convert_charfield_value)
         elif internal_type == 'UUIDField':
             converters.append(self.convert_uuidfield_value)
         return converters
 
+    def convert_charfield_value(self, value, expression, connection):
+        if isinstance(value, bytes):
+            return force_text(value)
+        return value
```

Comment:
I guess this does follow other existing methods, but:
```python
return force_text(value) if isinstance(value, bytes) else value
```

Reply:
Thanks. I updated this line.